scripts/pre_redeploy.sh

```diff
@@ -137,8 +137,9 @@ glance_registry_rolling: '100%'
 EOF
 
 # RLM-1438 patch openstack_hosts fork into 14.7.0 to ensure RLM-322 works
-if [[ ${RPC_TARGET_CHECKOUT} == "r14.7.0" ]]; then
+if [ ${RPC_TARGET_CHECKOUT} = "r14.7.0" -a ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/patch-rpc-o-14.7.0-ansible-role-requirements.complete" ]; then
```

Comment:
I could be wrong, but I think `${RPC_TARGET_CHECKOUT}` needs to be quoted if you are using single  bracket test notation.

Reply:
Nope but it's best practice, let me add it. It's important once the value is not predefined